src/AccountManager.ts

```diff
@@ -11,8 +11,18 @@ import {
 } from './typedRequestData.utils';
 
 export default class AccountManager {
+  private static instance: AccountManager;
+
   private _accounts: Wallet[] = [];
 
+  public static getInstance(): AccountManager {
```

Comment:
I think we should create a private constructor to avoid the creation of multiple instances

Reply:
updated